wallet/src/Pos/Wallet/Web/Account.hs

```diff
@@ -157,7 +157,7 @@ deriveAccountSK
 deriveAccountSK passphrase AccountId{..} accIndex = do
     key <- getSKById aiWId
     maybeThrow badPass $
-        deriveLvl2KeyPair passphrase key aiIndex accIndex
+        deriveLvl2KeyPair undefined passphrase key aiIndex accIndex
```

Comment:
Probably we may add `(MonadSlots, MonadGState)` to `AccountMode` to be able to provide `IsBootstrapEra` here.